code/pizzabot.js

```diff
@@ -1,7 +1,65 @@
 const vegetarian = "Vegetarian Pizza"
 const hawaiian = "Hawaiian Pizza"
 const pepperoni = "Pepperoni Pizza"
+const margherita = "Margherita Pizza"
 
 const pizzaPrice = 80
 
-//Put your Javscript code here:
+const pizzaVegImg = document.getElementById("veg")
+const pizzaHawImg = document.getElementById("haw")
+const pizzaPepImg = document.getElementById("pep")
+const pizzaMarImg = document.getElementById("mar")
+
+
+console.log(vegetarian, hawaiian, pepperoni, pizzaPrice)
```

Comment:
Before pushing your code to github, always check if there are any console.log() and delete them - it's considered bad practise to push them. Everyone can see your code out here and it's pretty messy